src/Stefna/Sniffs/Functions/MultiLineFunctionDeclarationSniff.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+
+namespace Stefna\Sniffs\Functions;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Standards\Squiz\Sniffs\Functions\MultiLineFunctionDeclarationSniff as BaseSniff;
+
+final class MultiLineFunctionDeclarationSniff extends BaseSniff
+{
+	public function processSingleLineDeclaration(File $phpcsFile, int $stackPtr, array $tokens): void
+	{
+		if ($this->isOneLineMethod($phpcsFile, $stackPtr, $tokens)) {
+			$scopeStart = $tokens[$stackPtr]['scope_opener'];
+			$scopeEnd = $tokens[$stackPtr]['scope_closer'];
+
+			// allow brackets on same line if body is empty
+			if ($scopeEnd === ($scopeStart + 1)) {
+				return;
+			}
+
+			if (($scopeEnd - $scopeStart) === 2 && $tokens[$scopeStart + 1]['code'] === T_WHITESPACE) {
+				$error = 'Whitespace not allowed between braces on empty method';
+				$fix = $phpcsFile->addFixableError($error, $scopeEnd, 'WhiteSpaceBetweenBraces');
+				if ($fix === true) {
+					$phpcsFile->fixer->replaceToken($scopeStart + 1, '');
+				}
+				return;
+			}
+		}
+
+		parent::processSingleLineDeclaration($phpcsFile, $stackPtr, $tokens);
+	}
+	/**
```

Comment:
Seems like this file needs formatting. Also, `@param array` is useless, since you don't have any hints of what this array could be